internal/namespaces/instance/v1/custom_server.go

```diff
@@ -354,6 +355,7 @@ func serverGetBuilder(c *core.Command) *core.Command {
 		for _, nic := range getServerResp.Server.PrivateNics {
 			pn, err := vpcAPI.GetPrivateNetwork(&vpc.GetPrivateNetworkRequest{
 				PrivateNetworkID: nic.PrivateNetworkID,
+				Zone:             args.Zone,
```

Comment:
IMHO: You should use getServerResp.Zone

Reply:
I was torn between the two options :sweat_smile: i'll update